practice-one/src/index.html

```diff
@@ -38,13 +35,50 @@
         <a class="btn btn-link btn-help-center" href="#">
           Help Center
         </a>
-
       </nav>
-
     </div>
-
   </header>
+  <!-- END HEADER -->
+
+  <!-- START MAIN -->
+  <main class="main-content">
+    <div class="container">
+      <!-- START SECTION GET-STARTED -->
+      <section class="section-get-started">
+        <section class="download">
+          <h1 class="heading-h1 download-heading">
+            Download app, save money, make friends!
+          </h1>
+          <p class="content download-content">
+            Its simple and its free. Play your part in reducing Carbon Footprint and help
+            Mother Nature to sustain its beauty. So what are you waiting for? Lets ride
+            together
+          </p>
 
+          <div class="download-links">
+            <a href="#" class="btn btn-link btn-download">
+              <svg class="download-logo" width="45" height="40">
+                <use xlink:href="./assets/images/download-logos.svg#logo-play-store"></use>
+              </svg>
+              Download
+            </a>
+            <a href="#" class="btn btn-link btn-download">
+              <svg class="download-logo" width="41" height="41">
+                <use xlink:href="./assets/images/download-logos.svg#logo-apple-store"></use>
+              </svg>
+              Download
+            </a>
+          </div>
+        </section>
+
+        <figure class="download-img">
```

Comment:
Should set the width and height for the image to avoid breaking the layout when the image can't load or when we change the image